src/pages/navigate/navigate.ts

```diff
@@ -120,7 +123,7 @@ export class NavigatePage {
 					var mrLevels = this.filterByProperty(questions, "mrLevel");
 					var a = mrLevels.map(f => {
 						var questionSet = questions.filter(s => s.mrLevel == f)
-						   .map(a => ({ text: a.questionText, questionId: a.questionId, questionStatus: this.findQStatus(a.currentAnswer, a.mrLevel) }));
+						   .map(a => ({ text: a.questionText, questionId: a.questionId, questionStatus: this.findQStatus(a.answers, a.mrLevel, a) }));
```

Comment:
looks great on the `All Questions page` answers all appear to be coming in correctly. :+1: